### DISCUSSION TITLE
How to debug Model for Index page

### DISCUSSION THREAD

**Discussion by AlexeyGorbachev** [OrchardCMS/OrchardCore#13243] [Q&A]
Hi All! How to debug Model for Index page? I need to debug modified Agency Theme. Where (file, class and function) is it possible to set breakpoint to see Model, which is sent to Index page?

**Comment by hishamco** [comment]
Agency theme is using Liquid syntax, could you please ellaborate what are you trying to debug?

**Reply by AlexeyGorbachev** [reply]
Thank you for your interest in my problem. I wanted simply to set a breakpoint on a server-site (in C# code) to view all fields of Model, which will be sent to Liquid page? Is it possible to do so?

**Reply by hishamco** [reply]
As I can see in this template there's no controllers, but you can use `ConsoleLog` ~~extension~~ if I remembered correctly to log the model fields & properties

Please have a look into https://github.com/sebastienros/fluid/tree/main/Fluid.MvcSample this will make it clear for you especially if you are using normal MVC

**Reply by hishamco** [reply]
```{{ Model.Content | console_log }}```

**Reply by AlexeyGorbachev** [reply]
I was unable to find ConsoleLog extension. What is the full name of ConsoleLog extension?

**Reply by hishamco** [reply]
It's a liquid filter

**Reply by AlexeyGorbachev** [reply]
Thank you. I got information from JSON Model as Text as an output from my Liquid template.,
